# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../clisp/bindings-glibc-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- clisp-2.33.2/modules/bindings/glibc/linux.lisp~	2004-03-10 18:52:06.000000000 +0100
+++ clisp-2.33.2/modules/bindings/glibc/linux.lisp	2005-01-03 02:52:22.634096240 +0100
@@ -2489,7 +2489,7 @@
 ;; miscellaneous signal stuff
 (def-call-out kill (:arguments (pid pid_t) (sig int)) (:return-type int))
 (def-call-out raise (:arguments (sig int)) (:return-type int))
-(def-call-out sigpause (:arguments (sig int)) (:return-type int))
+(def-call-out sigsuspend (:arguments (sig int)) (:return-type int))
 (def-call-out killpg (:arguments (pgrp pid_t) (sig int))
   (:return-type int))
 
